src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
```

Comment:
Didn't find Main implementation
In the main() method we need to create an array of figures (the size of the array can be 3 or 6, it doesn't matter). The first half of figures in this array should be generated with random parameters.